apps/astarte_trigger_engine/lib/astarte_trigger_engine/amqp_consumer/amqp_events_consumer.ex

```diff
@@ -26,15 +26,36 @@ defmodule Astarte.TriggerEngine.AMQPEventsConsumer do
   alias AMQP.Exchange
   alias AMQP.Queue
   alias Astarte.TriggerEngine.Config
+  alias Astarte.TriggerEngine
+  alias Astarte.TriggerEngine.Policy.PolicySupervisor
+  alias Astarte.Core.Triggers.Policy
 
   @connection_backoff 10000
-
-  @consumer Config.events_consumer!()
-
   # API
-
   def start_link(args \\ []) do
-    GenServer.start_link(__MODULE__, args, name: __MODULE__)
+    with {:ok, realm_name} <- Keyword.fetch(args, :realm_name),
```

Comment:
Let's consider if this module can be replaced by the newly found `ex_rabbit_pool`